go/libraries/doltcore/merge/merge_noms_rows.go

```diff
@@ -40,6 +42,76 @@ type rowMerger func(ctx context.Context, nbf *types.NomsBinFormat, sch schema.Sc
 
 type applicator func(ctx context.Context, sch schema.Schema, tableEditor editor.TableEditor, rowData types.Map, stats *MergeStats, change types.ValueChanged) error
 
+// mergeNomsTable merges a Noms table, which includes updating row data, secondary index data, and applying the specified |mergedSch|.
+func mergeNomsTable(ctx context.Context, tm *TableMerger, mergedSch schema.Schema, vrw types.ValueReadWriter, opts editor.Options) (*doltdb.Table, *MergeStats, error) {
```

Comment:
No code changes in this function; it was just extracted from `merge_rows.go` to keep the noms-specific code and prolly-specific code separate.